src/nti/contentfragments/interfaces.py

```diff
@@ -571,3 +576,16 @@ class ITagField(IPlainTextLineField):
 
     .. versionadded:: 1.2.0
     """
+
+class IAllowedAttributeProvider(interface.Interface):
+    """
+    A way to provide a whitelist of attribute names that would be allowed
+    while parsing a content fragment.
+
+    .. versionadded:: 1.3.2
+    """
+
+    allowed_attributes = IndexedIterable(title=u"An iterable of attribute names allowed in a particular context",
+                                         value_type=TextLine(title=u"The attribute name"),
```

Comment:
I'm not sure those are TextLines. They look like NativeStringLines to me.

This should probably become version 1.4.